builtin/aws/ecs/platform.go

```diff
@@ -961,6 +961,11 @@ func (p *Platform) Launch(
 	if !newListener {
 		def := listener.DefaultActions
 
+		// Direct all traffic to the new target group;
+		// effectively, removing traffic from any pre-existing
+		// forwarding default actions
+		tgs[0].Weight = aws.Int64(100)
```

Comment:
not too sure what the expected UX should be here 😅 if a new target group is created for the purpose of the deployment, would make sense to send traffic its way..however, how much, and how should it affect already configured target groups in the ALB listener?

Reply:
I noticed the same thing, this causes the need to double deploy, and I had the idea to use the Releaser in #794